LayoutTests/platform/ios/TestExpectations

```diff
@@ -3600,7 +3600,6 @@ webkit.org/b/237694 [ Debug ] webgl/1.0.3/conformance/uniforms/uniform-samplers-
 
 # image-set failing only on iOS
 webkit.org/b/253095 imported/w3c/web-platform-tests/css/css-images/image-set/image-set-type-rendering-2.html [ ImageOnlyFailure ]
-webkit.org/b/253095 imported/w3c/web-platform-tests/css/css-images/image-set/image-set-type-first-match-rendering.html [ ImageOnlyFailure ]
```

Comment:
Can we remove the expectation from LayoutTests/TestExpectations as well? (Not just the iOS one)

https://searchfox.org/wubkat/rev/b39f7b61225c26bb90158a196fc0e1e1b54b0880/LayoutTests/TestExpectations#3358

Reply:
Whoops. Yes.